src/platform/web/ui/session/settings/SettingsView.js

```diff
@@ -93,10 +93,13 @@ export class SettingsView extends TemplateView {
                 ]);
             })
         );
-
+        
         settingNodes.push(
             t.h3("Preferences"),
             row(t, vm.i18n`Scale down images when sending`, this._imageCompressionRange(t, vm)),
+            t.if(vm => vm.activeTheme, (t, vm) => {
+                return !import.meta.env.DEV? row(t, vm.i18n`Use the following theme`, this._themeOptions(t, vm)): null;
```

Comment:
You could put the `!import.meta.env.DEV` in the binding, e.g. `!import.meta.env.DEV && vm.activeTheme`, so the whole thing does not render on dev mode